apps/blog/serializers/image_serializer.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from apps.blog.models.image import Image
+
+class ImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Image
+        fields = '__all__'
```

Comment:
Name the fields here please, it is always good practice to list the field and declare as read only the ones not necessary or required during post or put requests.

```python

# ....
fields = [ 'all', 'the', 'fields', 'name',]